hw2/style-hw2.css

```diff
@@ -0,0 +1,214 @@
+}
+
+body {
+	background-color: #f9f9f9;
+}
+/*----------header----------*/
+.container {
+	width: 75%;
+	background-color: #fff;
+	margin: 0 auto;
+	box-shadow: 0 0 3px 2px #dbdbdb;
+	padding: 50px 45px 60px 45px;
+}
+
+.main-menu {
+	margin-top: 35px;
+	height: 40px;
+	width: 100%;
+	background: linear-gradient(to top, #444343, #5e5e5e);
+}
+
+.main-menu__item {
+	list-style: none;
+	float: left; 
+	border-right: rgba(255,255,255,0) 1px solid;
+	border-left: rgba(255,255,255,0) 1px solid;
+}
+
+.main-menu__link {
+	color: #ffffff;
+	text-decoration: none;
+	font-family: sans-serif;
+	display: block;
+	height: 100%;
+	width: 100px;
+	padding: 10px 25px;
+} 
+
+.main-menu__item:hover {
+	border-right: rgba(255,255,255,1) 1px solid;
+	border-left: rgba(255,255,255,1) 1px solid;
+	background: linear-gradient(to top, #2f2f2f, #4a4a4a);
+}
+/*----------aside----------*/
+.products {
+	margin-top: 40px;
+	margin-right: 40px;
+	width: 25%;
+	display: inline-block;
+	border: 1px solid #c8c8c8;
+}
+
+
+.products__item {
+	list-style: none;
+	border-bottom: 1px solid #c8c8c8;
+	color: #807e7e;
+}
+
+.products__item:last-child {
+	border-bottom: 0px solid #c8c8c8;
+}
+
+.products__link{
+	color: #807e7e;
+	text-decoration: none;
+	font-family: sans-serif;
+	display: block;
+	height: 100%;
+	padding: 15px 25px;
+} 
+
+.products__item:hover {
+	background-color: #f9f9f9;
+} 
+
+.products__item:hover .products__link {
+	color: #000;
+	padding-left: 35px;
+} 
+/*----------main----------*/
+.main-info {
+	width: 69%;
+	margin-top: 40px;
+	vertical-align: top;
+	overflow: hidden;
+	float: right; 
+}
+
+.main-info__item {
+	border: 1px solid #c8c8c8;
+	height: 400px;
+	position: relative
+}
+
+.main-info__image {
+	margin: auto;
+	position: absolute;
+  top: 0; left: 0; bottom: 0; right: 0;
+}
+
+
+.main-info__text {
+	padding: 35px 25px 60px 25px;
+	font-family: sans-serif;
+	color: #2c2c2c;
+	line-height: 150%;
+}
+
+.clear {
+	clear: both;
+}
+/*----------footer----------*/
+.main-footer {
+	height: 40px;
+	width: 100%;
+	background-color: #e1e1e1;
+}
+
+.footer-menu__item {
+	list-style: none;
+	display: block;
+	float: left; 
+}
+
+.footer-menu__link {
+	color: #2c2c2c;
+	text-decoration: none;
+	font-family: sans-serif;
+	font-size: 90%;
+	display: block;
+	height: 100%;
+	width: 100px;
+	padding: 10px 25px;
+}
+
+.footer-menu__item:hover .footer-menu__link  {
+	text-decoration: underline;
+} 
+
+.copyright {
+	padding: 10px 25px;
+	float: right;
+}
+/*----------header-flex----------*/
+.main-menu-flex {
+	margin-top: 35px;
+	height: 40px;
+	width: 100%;
+	background: linear-gradient(to top, #444343, #5e5e5e);
+}
+
+.main-menu-flex__list {
+	display:flex;
+}
+
+.main-menu-flex__item {
+	list-style: none;
+	border-right: rgba(255,255,255,0.0) 1px solid;
+	border-left: rgba(255,255,255,0.0) 1px solid;
+}
+
+.main-menu-flex__link {
+	color: #ffffff;
+	text-decoration: none;
+	font-family: sans-serif;
+	display: block;
+	height: 100%;
+	width: 100px;
```

Comment:
bad idea, don't use fixed width. Your site might have a lot of translations, so it would be better if menu items allow this